File: Selenium/Activities/Activity3.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        //Activity1
        driver.get("https://v1.training-support.net");
        System.out.println("Title of webpage:"+ driver.getTitle());
        driver.findElement(By.xpath("//a[text()='About Us']")).click();
        System.out.println("Title of the webpage now:" + driver.getTitle());

        //Activity2
        driver.get("https://v1.training-support.net/selenium/login-form");
        System.out.println("Navigated to webpage:"+driver.getTitle());
        driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
        driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");
        driver.findElement(By.xpath("//button[text()='Log in']")).click();
        if(driver.findElement(By.xpath("//div[@id='action-confirmation']")).isDisplayed()){
            System.out.println("Login successful!");
        }
        driver.close();

    }
}

File: Selenium/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:

    # Activity1 with XPATH
    driver.get("https://v1.training-support.net/")
    print("Title of the webpage:" + driver.title)
    driver.find_element(By.XPATH, "//a[text()='About Us']").click()
    print("Title of the webpage now:" + driver.title)

    # Activity2 with XPATH
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("Navigate to webpage:" + driver.title)
    driver.find_element(By.XPATH, "//input[@id='username']").send_keys("admin")
    driver.find_element(By.XPATH, "//input[@id='password']").send_keys("password")
    driver.find_element(By.XPATH, "//button[text()='Log in']").click()
    if (driver.find_element(By.XPATH, "//div[@id='action-confirmation']")).is_displayed():
        print("Login successful!")
    driver.close()
